src/StakeVault.sol

```diff
@@ -2,7 +2,8 @@
 
 pragma solidity ^0.8.26;
 
-import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
```

Comment:
If is not upgradable, just a clone, does it need to use OwnableUpgradable?

Reply:
Yeaa... I know this seems unexpected. I tried to clarify this with a comment in L:17 @3esmit

Reply:
Well, if is indeed necessary...